### PR TITLE
AN-130: Skip authorization checks if authorizers is empty array

### PR DESCRIPTION
**Pull request by acenolaza**
Fix: 
Skips calling AirnodeRrp.checkAuthorizationStatuses when authorizers array is empty and returns true for all pending requests.

Operation package commands should be back to working condition.

@Siegrift Please checkout the branch and run the e2e locally. For some weird reason requester-fulfill.feature tests started to pass automagically. 

We can also work on a separate branch for fixing e2e since this is actually for fixing public authorizers.

### CODE REVIEW

File: packages/node/src/evm/authorization/authorization-fetching.ts [hunk 1]
```diff
@@ -109,6 +109,14 @@ export async function fetch(
     return [[], {}];
   }
 
+  // If there are no authorizer contracts then endpoint is public
+  if (isEmpty(fetchOptions.authorizers)) {
```
Comment (andreogle) [inline packages/node/src/evm/authorization/authorization-fetching.ts:113]: Nice and simple 👍🏻

**Review by andreogle — APPROVED** [review]

File: packages/node/src/evm/authorization/authorization-fetching.ts (hunk 1 above)
Comment (Siegrift) [reply packages/node/src/evm/authorization/authorization-fetching.ts:113]: Actually a nice trick to convey the same information is to create a very specific function/variable for that:
```js
const isEndpointPublic = isEmpty(fetchOptions.authorizers)
if (isEndpointPublic) ...
```

Ofc, this is perfectly fine - but I've read this tip from a smartass from some book :) :D

**Review by Siegrift — APPROVED** [review]
:1st_place_medal: I've tried running unit tests locally (because they failed on CI) but they pass locally for me so I retried them on CI...

The e2e tests still fail for me locally and on CI as well. I'll try to debug them as part https://api3dao.atlassian.net/browse/AN-177 :)

File: packages/node/src/evm/authorization/authorization-fetching.ts (hunk 1 above)
Comment (acenolaza) [reply packages/node/src/evm/authorization/authorization-fetching.ts:113]: I really like that idea but I think I like consistency more tbh 😄

**Comment by acenolaza** [comment]
>  I've tried running unit tests locally (because they failed on CI) but they pass locally for me so I retried them on CI...
> 
> The e2e tests still fail for me locally and on CI as well. I'll try to debug them as part https://api3dao.atlassian.net/browse/AN-177 :)

There is also a "Check documentation" also failing. I'm going to look at it later.

Thanks @Siegrift for taking care of the e2e! I really appreciate your help.
Also thank you @andreogle for the patience and the attention to detail when reviewing PRs.

**Pull request merged**